src/main/java/frc/robot/subsystems/roller/Roller.java

```diff
@@ -0,0 +1,65 @@
+package frc.robot.subsystems.roller;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.hardware.digitalinput.DigitalInputInputsAutoLogged;
+import frc.robot.hardware.digitalinput.IDigitalInput;
+import frc.robot.hardware.motor.IMotor;
+import frc.utils.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+public class Roller extends GBSubsystem {
+
+	private final IMotor motor;
+	private final IDigitalInput digitalInput;
+	private final DigitalInputInputsAutoLogged digitalInputsInputs;
+	private final RollerStuff rollerStuff;
```

Comment:
is there any reason some of the stuff of the roller stuff are saved in vars but the others kept in the rollerStuff object?